cmd/kubelet/app/options/options.go

```diff
@@ -405,6 +404,8 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	fs.MarkDeprecated("really-crash-for-testing", "will be removed in a future version.")
 	fs.Float64Var(&f.ChaosChance, "chaos-chance", f.ChaosChance, "If > 0.0, introduce random client errors and latency. Intended for testing.")
 	fs.MarkDeprecated("chaos-chance", "will be removed in a future version.")
+	fs.StringVar(&f.SeccompProfileRoot, "seccomp-profile-root", f.SeccompProfileRoot, "<Warning: Alpha feature> Directory path for seccomp profiles.")
+	fs.MarkDeprecated("seccomp-profile-root", "will be removed in a future version.")
```

Comment:
Indicate what the behavior will be when this flag is removed, and the specific version the flag is being removed in:

```suggestion
	fs.MarkDeprecated("seccomp-profile-root", "will be removed in 1.23, in favor of using the `<root-dir>/seccomp` directory")
```